### PR TITLE
add error message feature

### PR DESCRIPTION
**Pull request by NOI-lgtm**
When user types wrong url then the input file is shown with a red highlighted color and gives warning message. 
screen shot 2018-08-26 at 20 21 35

If the the icons are not downloaded or it does not exist in the given  url then it shows the message with icons not found.
screen shot 2018-08-26 at 20 29 27

### CODE REVIEW

File: static/js/app.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const vm = new Vue({
   data: {
     url: null,
     loading: false,
+    error: false,
+    completed: true,
```
Comment (deepredsky) [inline static/js/app.js]: this should be false at first

Comment (NOI-lgtm) [reply static/js/app.js]: fixed completed true to false

**Comment by nirmalathapa** [comment]
screen shot 2018-08-26 at 20 21 35
screen shot 2018-08-26 at 20 29 27

File: index.html [hunk 2]
```diff
@@ -15,14 +15,15 @@
...
+                <h1 class="title main">Icon Extractor</h1>
                 <div class="columns is-centered">
                   <div v-if="loading" class="loader">
                   </div>
                   <div v-else>
                     <div class="field is-grouped">
                       <div class="control">
-                        <input class="input is-primary" type="text" placeholder="Insert website url" v-model="url ">
+                        <input class="input is-primary" type="text" placeholder="Insert website url" v-model="url " v-bind:class="{'is-danger':error}">
+                        <p class="help is-danger" v-if="error">This link is invalid</p>
```
Comment (deepredsky) [inline index.html]: help should be outside control

**Review by deepredsky — APPROVED** [review]

**Pull request merged**